pubspec.yaml

```diff
@@ -45,20 +45,28 @@ dependencies:
   timeline_tile: ^2.0.0
   sliver_tools: ^0.2.7
 
-flutter_native_splash:
-  image: assets/non-free/logo_payoff.png
-  color: "F7F7F7"
-  android_disable_fullscreen: true
-
 dev_dependencies:
   flutter_test:
     sdk: flutter
   integration_test:
     sdk: flutter
-  json_serializable: ^4.1.4
+  json_serializable: ^6.0.0
```

Comment:
For some reason the `./codegen.sh` script doesn't work for me anymore now. I think it is related to this major version bump. Can you check that?

Reply:
Yes it looks like i broke it. I downgraded the packages now. Btw, should i add the generation of the app icon to the codegen.sh as well?